src/test/java/seedu/sellsavvy/logic/commands/ordercommands/DeleteOrderCommandTest.java

```diff
@@ -0,0 +1,101 @@
+package seedu.sellsavvy.logic.commands.ordercommands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.sellsavvy.logic.commands.personcommands.PersonCommandTestUtil.assertCommandFailure;
+import static seedu.sellsavvy.logic.commands.personcommands.PersonCommandTestUtil.assertCommandSuccess;
+import static seedu.sellsavvy.testutil.TypicalIndexes.INDEX_FIRST_ORDER;
+import static seedu.sellsavvy.testutil.TypicalIndexes.INDEX_SECOND_ORDER;
+import static seedu.sellsavvy.testutil.TypicalPersons.getTypicalAddressBook;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.sellsavvy.commons.core.index.Index;
+import seedu.sellsavvy.logic.Messages;
+import seedu.sellsavvy.model.Model;
+import seedu.sellsavvy.model.ModelManager;
+import seedu.sellsavvy.model.UserPrefs;
+import seedu.sellsavvy.model.order.Order;
+import seedu.sellsavvy.model.person.Person;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for
+ * {@code DeleteOrderCommand}.
+ */
+public class DeleteOrderCommandTest {
```

Comment:
There should be a test case where no `Person`'s orders are listed when executing this command (where we should be expecting the command to fail).
This can be achieved by not updating the selected person using `updateSelectedPerson(Person)` and attempting to execute the command.

Reply:
This wasn't handled in first place.

Reply:
now that it's handled, shouldn't there be a test case now? @yu-sutong